js/coinbin.js

```diff
@@ -1186,6 +1221,33 @@ $(document).ready(function() {
 		});
 	}
 
+	// broadcast transaction via blockexplorer.com (testnet)
+	function rawSubmitbexp_bitcointestnet(thisbtn){ 
+		$(thisbtn).val('Please wait, loading...').attr('disabled',true);
+		var sendtr = {'tx':$("#rawTransaction").val()};
+		$.ajax ({
+			type: "POST",
+			url: "https://testnet.blockexplorer.com/api/tx/send",
+			data: {'rawtx':$("#rawTransaction").val()},
+			error: function(res) {
+				var r = ' ';
+				r += (res.responseText) ? res.responseText : '';
+				r = (r!='') ? r : ' Failed to broadcast'; // build response 
+				$("#rawTransactionStatus").addClass('alert-danger').removeClass('alert-success').removeClass("hidden").html(r).prepend('<span class="glyphicon glyphicon-exclamation-sign"></span>');
+			},
+            success: function(res) {
+				if(res.txid){
+					$("#rawTransactionStatus").addClass('alert-success').removeClass('alert-danger').removeClass("hidden").html(' Txid: '+res.txid+'<br> Check this transaction by searching Txid on <a href="https://testnet.blockexplorer.com/">testnet.blockexplorer.com</a>');
```

Comment:
This change should automatically give the user a link directly to their transaction on the blockchain explorer (you might want to test it):

```suggestion
					$("#rawTransactionStatus").addClass('alert-success').removeClass('alert-danger').removeClass("hidden").html(' TXID: '+res.txid+'<br> <a href="https://testnet.blockexplorer.com/tx/'+res.txid+'" target="_blank">View on Blockchain Explorer</a>');
```